setup.py

```diff
@@ -0,0 +1,42 @@
+"""Set up sshprank."""
+import os
+
+from setuptools import find_packages, setup
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(here, "README.md"), encoding="utf-8") as readme:
+    long_description = readme.read()
+
+
+setup(
+    name="sshprank",
+    version="1.2.3",
+    description="SSH mass-scanner, login cracker and banner grabber tool",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url="https://github.com/noptrix/sshprank",
+    author="noptrix",
+    author_email="noptrix@nullsecurity.net",
+    maintainer="Fabian Affolter",
+    maintainer_email="fabian@affolter-engineering.ch",
+    license="MIT",
+    install_requires=["paramiko", "shodan", "shodan", "python-masscan", "click"],
```

Comment:
i think we can remove one "shodan" :) also, is "click" needed for the setup.py? for sshprank it is not. can you please check that

Reply:
Yes, that's a duplicate. 

`shodan` should pull in `click` but I got an error message and adding it solved it. It's indirect dependency.